boards/ti/lp_mspm0g3507/doc/index.rst

```diff
@@ -0,0 +1,183 @@
+
+* Analog.
+
+  * Two 12-bit 4-Msps ADCs.
+
+  * Configurable internal shared voltage reference.
+
+  * One 12-bit 1-Msps DAC.
+
+  * Three high speed comparators with built-in reference DACs.
+
+  * Two zero-drift zero-crossover op-amps with programmable gain.
+
+* Digital.
+
+  * Two 16-bit advanced control timers.
+
+  * Five general-purpose timers.
+
+    * One 16-bit general-purpose timer for QEI interface.
+
+    * One 32-bit high resolution general-purpose timer.
+
+    * Two 16-bit timers with deadband support and up to 12 PWM Channels.
+
+  * Two windowed-watchdog timers.
+
+  * One RTC with alarm and calendar modes.
+
+* Data Integrity and Encryption.
+
+  * One AES HW accelerator capable of CTR, CBC, and ECB modes.
+
+  * One Cyclic Redundancy Check (CRC) accelerator.
+
+  * One True Random Number Generator (TRNG).
+
+* Communication.
+
+  * Four UARTs, one with support for advanced modes such as LIN and Manchester.
+
+  * Two I2C supporting SMBUS/PMBUS and speeds up to FM+ (1Mbits/s).
+
+  * Two SPI, one with max speed 32Mbits/s.
+
+  * One CAN interface supporting CAN 2.0 A or B and CAN-FD.
+
+.. image:: img/lp_mspm0g3507.webp
+     :align: center
+     :alt: MSPM0G3507 LaunchPad development board
+
+Zephyr uses the ``lp_mspm0g3507`` board for building LP_MSPM0G3507
+
+Features:
+*********
+
+- Onboard XDS110 debug probe
+- EnergyTrace technology available for ultra-low-power debugging
+- 2 buttons, 1 LED and 1 RGB LED for user interaction
+- Temperature sensor circuit
+- Light sensor circuit
+- External OPA2365 (default buffer mode) for ADC (up to 4 Msps) evaluation
+- Onboard 32.768-kHz and 40-MHz crystals
+- RC filter for ADC input (unpopulated by default)
+
+Details on the MSPM0G3507 LaunchPad can be found on the `TI LP_MSPM0G3507 Product Page`_.
+
+Supported Features
+==================
+
+.. zephyr:board-supported-hw::
+
+The Zephyr ``lp_mspm0g3507`` board target supports the following hardware
+features:
+
++-----------+------------+-----------------------+
+| Interface | Controller | Driver/Component      |
++===========+============+=======================+
+| NVIC      | on-chip    | nested vectored       |
+|           |            | interrupt controller  |
++-----------+------------+-----------------------+
+| SYSTICK   | on-chip    | system clock          |
++-----------+------------+-----------------------+
+| UART      | on-chip    | serial                |
++-----------+------------+-----------------------+
+| CLOCKMUX  | on-chip    | clockctl              |
++-----------+------------+-----------------------+
+| PINMUX    | on-chip    | pinctrl               |
++-----------+------------+-----------------------+
+| GPIO      | on-chip    | gpio                  |
++-----------+------------+-----------------------+
+
+More details about the supported peripherals are available in `MSPM0G3507 TRM`_.
+Other hardware features are not currently supported by the Zephyr kernel.
```

Comment:
```suggestion
```

Reply:
Should remove this part? If so may I know the reason?

Reply:
the hw tag does this, so this part needs to go

Reply:
thanks, done